lstchain/irf/hdu_table.py

```diff
@@ -316,8 +307,8 @@ def create_event_list(data, run_number, source_name):
 
     ev_header["RA_PNT"] = tel_pnt_sky_pos.ra.value
     ev_header["DEC_PNT"] = tel_pnt_sky_pos.dec.value
-    ev_header["ALT_PNT"] = round(np.rad2deg(data["pointing_alt"].value.mean()), 6)
-    ev_header["AZ_PNT"] = round(np.rad2deg(data["pointing_az"].value[0]), 6)
+    ev_header["ALT_PNT"] = np.rad2deg(data["pointing_alt"].value.mean())
```

Comment:
why mean for one and [0] for the other? 

Also, better use `to_value(u.deg)` instead of `np.rad2deg(...value)`

Reply:
For AZ_PNT, the values were very close, so no point in using mean().
These header values have to be without units, but the index tables can have the units(most probably). Gammapy adds the units while using these values.
And so, to convert the radians to degree, using the numpy function seemed easier.

Reply:
No, please always use `.to_value(unit)` when converting astropy quanties to numbers without units. That always works, regardless of which unit the quantity has.

Reply:
Corrected it accordingly